client/src/components/Carousel.jsx

```diff
@@ -8,37 +8,67 @@ import CarouselItem from './CarouselItem.jsx';
 const ItemContainer = styled.div`
   display: flex;
   flex-direction: row;
-  justify-content: space-around;
+  margin: 0 0 20px 20px;
   width: 100%;
   height: 30%;
   overflow: hidden;
 `
 const Thumbnail = styled.div`
-  width: 15%;
-  margin: 1%;
+  flex: 1 0 100%;
```

Comment:
Is the flex property what you use to achieve the horizontal positioning of all the videos in the carousel? Did not know about flex at all so this is great to know.

Reply:
This particular use of 'flex' actually controls the grow/shrink ratio of a given item relative to the other items in the container, but yes! `display: flex` and `flex-direction: row` do control the row-styled appearance of all the items in my carousel. Flexbox rules for one-dimensional rendering like this.